king_phisher/client/application.py

```diff
@@ -411,6 +428,9 @@ def theme_file(self):
 			return DISABLED
 		return find.find_data_file(os.path.join('style', self._theme_file))
 
+	def do_config_load(self):
+		self.load_config(load_defaults=True)
+
 	def load_config(self, load_defaults=False):
```

Comment:
This should become the new `do_config_load` method. Existing references to `load_config` should either A) be updated to `.emit('config-load')` or B) `load_config` should simply emit the signal. I would prefer solution A unless there are a lot of occurrences where the config is explicitly loaded (I don't think there are).